src/Nav.jsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom"
+
+export const Nav = () => {
+    const username = 'mallionaire'
+    return (
+        <nav>
+            <ul className="nav">
+                <li> Categories: </li>
+                <Link to={`/reviews?category=strategy`}><li> Strategy </li></Link>
+                <Link to={'/reviews?category=hidden-roles'}><li> Hidden-Roles </li></Link>
+                <Link to={`/reviews?category=dexterity`}><li>Dexterity</li></Link>
+                <Link to={`/reviews?category=push-your-luck`}><li>Push-Your-Luck</li></Link>
+                <Link to={`/reviews?category=roll-and-write`}><li>Roll-and-Write</li></Link>
+                <Link to={`/reviews?category=deck-building`}><li>Deck-Building</li></Link>
+                <Link to={`/reviews?category=engine-building`}><li>Engine-Building</li></Link>
+                <Link to={`/${username}/comments`}><li>My Comments</li></Link>
```

Comment:
When we get to looking at comments you will see that the comments are attached to a review rather than a user